src/Billing/Claim.php

```diff
@@ -241,8 +241,11 @@ public function loadPayerInfo(&$billrow)
 
             $ins = count($this->payers);
             if (
-                ($drow['provider'] == $billrow['payer_id'] || $billrow['payer_id'] == null) &&
-                empty($this->payers[0]['data'])
+                (
```

Comment:
It may improve reading clarity to abstract this conditional chain into a small function.